app/api/api_v1/routers/search.py

```diff
@@ -66,12 +56,6 @@ def _search_request(db: Session, search_body: SearchRequestBody) -> SearchRespon
             req=_get_browse_args_from_search_request_body(search_body),
         )
     else:
-        if search_body.keyword_filters is not None:
```

Comment:
Not entirely clear why this has been removed :shrug:

Reply:
please check for sanity, but my reading of the code was that was only necessary when updating the keywrod filters to process the categories, hence the whole block is no longer needed...

Reply:
Yeah I follow now - looks good to remove :smile: